package.json

```diff
@@ -23,16 +23,16 @@
     "minimist": "^1.2.3",
     "mustache": "^4.0.0",
     "node-html-parser": "^4.1.5",
-    "nodegit": "^0.27.0",
-    "rimraf": "^3.0.2"
+    "rimraf": "^3.0.2",
+    "simple-git": "^2.48.0"
   },
   "devDependencies": {
     "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-prettier": "^3.4.1",
-    "husky": "^7.0.0",
-    "lint-staged": "^11.1.2",
+    "husky": "^7.0.4",
```

Comment:
Not sure why these got changed, but I can roll it off if you want

Reply:
Yep, let's remove those changes for this PR

Reply:
Done